tests/e2e/test_target.go

```diff
@@ -0,0 +1,221 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+type ExecutionTarget interface {
+	GetTargetType() string
+	GetLogs(path string) []byte
+	GetTestScriptPath(isConsumer bool, script string) string
+	ExecCommand(name string, arg ...string) *exec.Cmd
+	ExecDetachedCommand(name string, args ...string) *exec.Cmd
```

Comment:
I understand, and maybe it's obvious, but could you comment here a sentence about what you expect from ExecCommand (return only after the command has been executed to completion) and ExecDetachedCommand (start the command, then return immediately)?